File: MartinAndOrlasWeddingApplication/WeddingServices/Implementation/Guest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WeddingServices.Interface;

namespace WeddingServices.Implementation
{
    public class Guest : IGuest
    {
        public int Id { get; set; }
        public string FoodChoice { set; get; }
        public string Email { get; set; }
        public string Firstname { get; set; }
        public string Surname { get; set; }
        public string DietComment { get; set; }
        public string Status { get; set; }
        public string GuestName { get; set; }

        public string getFirstname()
        {
            return Firstname;
        }

        public string getSurname()
        {
            return Surname;
        }

        public string getGuestName()
        {
            return GuestName;
        }

        public string getDietComment()
        {
            return DietComment;
        }

        public string getStatus()
        {
            return Status;
        }

        public int getIdentifier()
        {
            return Id;
        }

    }
}

File: BigDayApplication/app/src/main/java/com/bearingpoint/bigdayapplication/tasks/Guest.java
package com.bearingpoint.bigdayapplication.tasks;

/**
 * Created by Martin.Sexton on 23/04/2016.
 */
public class Guest {
    private int identifier;
    private String firstname;
    private String surname;
    private String status;
    private String guestName;
    private String dietComment;

    public int getIdentifier(){return identifier;}
    public String getFirstname(){return firstname;}
    public String getSurname(){return surname;}
    public String getStatus(){return status;}
    public String getGuestName(){return guestName;}
    public String getDietComment(){return dietComment;}

    public void setIdentifier(int id){identifier = id;}
    public void setFirstname(String fn){firstname=fn;}
    public void setSurname(String sn){surname=sn;}
    public void setStatus(String s){status=s;}
    public void setGuestName(String gn){guestName=gn;}
    public void setDietComment(String d){dietComment=d;}

    public String toString(){
        return getFirstname() + " " + getSurname() + " has " + getStatus();
    }
}
